### PR TITLE
add new addon SupremeExpansion | RelativoBR

### PR DESCRIPTION
**Pull request by RelativoBR**

### CODE REVIEW

File: pages/Addons.md [hunk 1]
```diff
@@ -72,6 +72,7 @@ SfChunkInfo | FN_FAL113 | <details><summary>expand</summary>Scan the amount of s
...
 Missile Warfare | Kaiyan2006 | <details><summary>expand</summary>Missile Warfare adds a bunch of missiles to Slimefun that can be fired at others!</details> | N/A | [GitHub](https://github.com/koiboi-dev/MissileWarfare) | [Releases](https://github.com/koiboi-dev/MissileWarfare/releases)
 SlimefunAdvancements | char321 | <details><summary>expand</summary>SlimefunAdvancements adds fully configurable advancements to Slimefun. Easily create your own progression!</details> | [Readme](https://github.com/qwertyuioplkjhgfd/SlimefunAdvancements/blob/main/README.md) | [Github](https://github.com/qwertyuioplkjhgfd/SlimefunAdvancements) | [Builds](https://thebusybiscuit.github.io/builds/qwertyuioplkjhgfd/SlimefunAdvancements/main/)
+SupremeExpansion | RelativoBR | <details><summary>expand</summary>Supreme Expansion is a addon to Slimefun 4 which brings various new resources to craft powerful contains items, gear and machines</details> | [Readme](https://github.com/RelativoBR/SupremeExpansion/blob/master/README.md) | [Github](https://github.com/RelativoBR/SupremeExpansion) | [Builds](https://thebusybiscuit.github.io/builds/RelativoBR/SupremeExpansion/master/)
```
Comment (Sefiraat) [inline pages/Addons.md:75]: `various new resources to craft powerful contains`

Looks like there was meant to be something between `powerful `and `contains`?

Comment (Sefiraat) [inline pages/Addons.md:75]: `https://thebusybiscuit.github.io/builds/RelativoBR/SupremeExpansion/master/`

404? Perhaps this has not been added to builds yet?

**Review by Sefiraat — CHANGES_REQUESTED** [review]

**Comment by Phoenix-Starlight** [comment]
``master`` is no longer the default, it is now named ``main``.

**Comment by Phoenix-Starlight** [comment]
Oh, guess I can't format via email. How unfortunate. ;-;

File: pages/Addons.md (hunk 1 above)
Comment (RelativoBR) [reply pages/Addons.md:75]: Yes

**Review by svr333 — APPROVED** [review]

**Comment by Phoenix-Starlight** [comment]
Craft powerful what? Items? (You need a noun)

**Comment by RelativoBR** [comment]
@Phoenix-Starlight  add new description,

**Comment by RelativoBR** [comment]
@Sefiraat can you tell me why I couldn't get approval to display the addon?

**Comment by Phoenix-Starlight** [comment]
In no order of listing:
1. 'tools e armor' has a typo.
2. 'generador' is also typo'ed.
3. Also 'new resources,  to craft' has been double spaced by accident.
4. 'which 100+ resouces' needs 'add' in between.
5. Remove 'contains' from 'contains weapons', it doesn't fit in with
the sentence.
6. Plurals everywhere: 'generators', 'capacitors', 'armors'

**Comment by RelativoBR** [comment]
close publish addon in wiki

**Comment by RelativoBR** [comment]
I don't have mastery of the grammar of the language, I'm using google translate, better choose not to publish, sorry for the inconvenience.

File: pages/Addons.md [hunk 2]
```diff
@@ -72,6 +72,7 @@ SfChunkInfo | FN_FAL113 | <details><summary>expand</summary>Scan the amount of s
...
 SlimefunAdvancements | char321 | <details><summary>expand</summary>SlimefunAdvancements adds fully configurable advancements to Slimefun. Easily create your own progression!</details> | [Readme](https://github.com/qwertyuioplkjhgfd/SlimefunAdvancements/blob/main/README.md) | [Github](https://github.com/qwertyuioplkjhgfd/SlimefunAdvancements) | [Builds](https://thebusybiscuit.github.io/builds/qwertyuioplkjhgfd/SlimefunAdvancements/main/)
+SupremeExpansion | RelativoBR | <details><summary>expand</summary>Supreme Expansion is a addon to Slimefun which 100+ various new resources,  to craft powerful items, contains weapons, tools e armor (titanium, aurum,  adamantium, thornium, magic, rare, epic, legendary and supreme), 12 new eletric gerador, 5 new capacitor, and more... </details> | [Readme](https://github.com/RelativoBR/SupremeExpansion/blob/main/README.md) | [Github](https://github.com/RelativoBR/SupremeExpansion) | [Builds](https://thebusybiscuit.github.io/builds/RelativoBR/SupremeExpansion/main/)
```
Comment (Sefiraat) [inline pages/Addons.md:75]: Supreme Expansion is a addon for Slimefun which adds 100+ various new resources that will allow you to craft powerful new items, weapons, tools and armor. These can be made up from titanium, aurum, adamantium, thornium with some being magical, rare, epic, legendary or supreme! It also adds 12 new electric generators, 5 new capacitors and even more...

Comment (Sefiraat) [reply pages/Addons.md:75]: Try this, assuming @Phoenix-Starlight is happy with this then we can get your addon in the wiki :)

**Review by Sefiraat — APPROVED** [review]

File: pages/Addons.md [hunk 3]
```diff
@@ -72,6 +72,7 @@ SfChunkInfo | FN_FAL113 | <details><summary>expand</summary>Scan the amount of s
...
 SlimefunAdvancements | char321 | <details><summary>expand</summary>SlimefunAdvancements adds fully configurable advancements to Slimefun. Easily create your own progression!</details> | [Readme](https://github.com/qwertyuioplkjhgfd/SlimefunAdvancements/blob/main/README.md) | [Github](https://github.com/qwertyuioplkjhgfd/SlimefunAdvancements) | [Builds](https://thebusybiscuit.github.io/builds/qwertyuioplkjhgfd/SlimefunAdvancements/main/)
+SupremeExpansion | RelativoBR | <details><summary>expand</summary>Supreme Expansion is a addon for Slimefun which adds 100+ various new resources that will allow you to craft powerful new items, weapons, tools and armor. These can be made up from titanium, aurum, adamantium, thornium with some being magical, rare, epic, legendary or supreme! It also adds 12 new electric generators, 5 new capacitors and even more... </details> | [Readme](https://github.com/RelativoBR/SupremeExpansion/blob/main/README.md) | [Github](https://github.com/RelativoBR/SupremeExpansion) | [Builds](https://thebusybiscuit.github.io/builds/RelativoBR/SupremeExpansion/main/)
```
Comment (svr333) [inline pages/Addons.md:75]:
```suggestion
SupremeExpansion | RelativoBR | <details><summary>expand</summary>Supreme Expansion is an addon for Slimefun which adds 100+ various new resources that will allow you to craft powerful new items, weapons, tools and armor. These can be made up from titanium, aurum, adamantium, thornium with some being magical, rare, epic, legendary or supreme! It also adds 12 new electric generators, 5 new capacitors and even more... </details> | [Readme](https://github.com/RelativoBR/SupremeExpansion/blob/main/README.md) | [Github](https://github.com/RelativoBR/SupremeExpansion) | [Builds](https://thebusybiscuit.github.io/builds/RelativoBR/SupremeExpansion/main/)
```

**Review by svr333 — APPROVED** [review]

**Review by Sefiraat — APPROVED** [review]

**Comment by Phoenix-Starlight** [comment]
Just a note: I would be happy as long as if you were to read it out
aloud it makes sense. I am not super strict about grammar but if it
sounds wrong or bad when you read it out aloud, then it needs fixing.
(Or you could use Grammarly if you have serious troubles with English)

**Pull request merged**